scripts/prettier/prettier-helpers.js

```diff
@@ -10,6 +10,21 @@ const prettierBin = getPrettierBinary();
 const prettierRulesConfig = path.join(repoRoot, 'prettier.config.js');
 const prettierIgnorePath = path.join(repoRoot, '.prettierignore');
 
+const prettierSupportedFileExtensions = [
```

Comment:
Can you export this list from somewhere and use it in the lint-staged config so it's not defined twice? As-is it's highly likely to get out of sync.